templates/header.php

```diff
@@ -4,8 +4,13 @@
 <head>
     <meta charset="<?php bloginfo('charset'); ?>" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <?php wp_head(); ?>
+    <?php
+    // must be before wp_head() to render CSS
+    $header = do_blocks('<!-- wp:template-part {"slug":"header"} /-->');
+    wp_head();
+    ?>
 </head>
 
 <body <?php body_class(); ?>>
-    <?php block_header_area();
\ No newline at end of file
+    <?php wp_body_open(); ?>
```

Comment:
According to the wiki: https://developer.wordpress.org/reference/functions/wp_body_open/
This was introduced in WP 5.2.0, and our readme says we support from WP 3+ (which man, lot changed since then). Would _maybe_ check before executing:
```
if (function_exists('wp_body_open')) {
     wp_body_open(); 
 }
``` 
Of course, `block_header_area()` wasn't supported until 5.9, so maybe we're out of the woods on this one.

Reply:
yah thanks, good callout. this file only gets invoked if the version is 5.9 above because it checks if `wp_is_block_theme` is a function and true, so i think we're ok

https://github.com/code4recovery/12-step-meeting-list/blob/main/includes/functions.php#L2386